### PR TITLE
Recurring application charge/customize

### PR DESCRIPTION
**Pull request by jcbarton**

### CODE REVIEW

File: resources/recurring-application-charge.js [hunk 1]
```diff
@@ -39,17 +39,16 @@ RecurringApplicationCharge.prototype.activate = function activate(id, params) {
 };
 
 /**
- * Customize a recurring application charge (increase the capped amount)
+ * Customize a recurring application charge.
  *
  * @param {Number} id Recurring application charge ID
  * @param {Number} cappedAmount The new capped amount to apply to the recurring application charge
```
Comment (PeterBrooksbank) [inline resources/recurring-application-charge.js]:
```suggestion
 * @param {Object} params Customization parameters
```

File: test/fixtures/recurring-application-charge/res/customize.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "return_url": "http://yourapp.com",
+        "billing_on": "2018-12-15",
+        "created_at": "2018-11-15T16:22:02-05:00",
+        "updated_at": "2018-11-15T16:23:29-05:00",
+        "test": null,
+        "activated_on": "2018-11-15",
+        "trial_ends_on": "2018-11-15",
+        "cancelled_on": null,
+        "trial_days": 0,
+        "capped_amount": "100.00",
```
Comment (PeterBrooksbank) [inline test/fixtures/recurring-application-charge/res/customize.json]:
```suggestion
        "capped_amount": "200.00",
```
Falls inline with the req amount

File: test/fixtures/recurring-application-charge/res/customize.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        "balance_used": 0.0,
+        "balance_remaining": 100.0,
```
Comment (PeterBrooksbank) [inline test/fixtures/recurring-application-charge/res/customize.json]:
```suggestion
        "balance_remaining": 200.0,
```

File: test/recurring-application-charge.test.js [hunk 4]
```diff
@@ -100,4 +100,19 @@ describe('Shopify#recurringApplicationCharge', () => {
     return resource.delete(455696195)
       .then(data => expect(data).to.deep.equal({}));
   });
+
+  it('customizes a recurring application charge', () => {
+    const input = fixtures.req.customize;
+    const output = fixtures.res.activate;
```
Comment (PeterBrooksbank) [inline test/recurring-application-charge.test.js]:
```suggestion
    const output = fixtures.res.customize;
```

Changed res fixture

File: test/fixtures/recurring-application-charge/res/customize.json (hunk 2 above)
Comment (jcbarton) [reply test/fixtures/recurring-application-charge/res/customize.json]: Interestingly, 200 in the request + 100 in response is what is part of the Shopify docs. I think the tests are built around these samples. You are right though - so I'll change

**Pull request merged**